contracts/upgradeable/mocks/OwnableMintableERC721MockUpgradeable.sol

```diff
@@ -0,0 +1,11 @@
+// SPDX-License-Identifier: Apache-2.0
+
+pragma solidity ^0.8.18;
+
+/// @dev This upgradeable mock smart contract is intended to be used with `@defi-wonderland/smock` and doesn't need any
+///     business logic.
+contract OwnableMintableERC721MockUpgradeable {
```

Comment:
This is probably the exact same as the non upgradeable so we should not need it.